deploy/helm/cht-chart/templates/couchdb-3-deployment.yaml

```diff
@@ -0,0 +1,83 @@
+{{- if .Values.couchdb.clusteredCouch_enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cht.service: couchdb-3
+  name: cht-couchdb-3
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cht.service: couchdb-3
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        cht.service: couchdb-3
+    spec:
+      tolerations:
+      - key: {{ .Values.toleration.key }}
+        operator: {{ .Values.toleration.operator }}
+        value: {{ .Values.toleration.value | quote }}
+        effect: {{ .Values.toleration.effect }}
+      containers:
+        - env:
+            - name: COUCHDB_LOG_LEVEL
+              value: error
+            - name: COUCHDB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: cht-couchdb-credentials
+                  key: COUCHDB_PASSWORD
+            - name: COUCHDB_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: cht-couchdb-credentials
+                  key: COUCHDB_SECRET
+            - name: COUCHDB_SYNC_ADMINS_NODE
+              valueFrom:
+                configMapKeyRef:
+                  name: couchdb-servers-configmap
+                  key: COUCHDB_SYNC_ADMINS_NODE
+            - name: COUCHDB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: cht-couchdb-credentials
+                  key: COUCHDB_USER
+            - name: COUCHDB_UUID
+              valueFrom:
+                secretKeyRef:
+                  name: cht-couchdb-credentials
+                  key: COUCHDB_UUID
+            - name: SVC_NAME
+              value: couchdb-3.{{ .Values.namespace }}.svc.cluster.local
+          image: public.ecr.aws/s5s3h4s7/cht-couchdb:{{ .Values.chtversion }}-{{ .Values.chtversion }}
+          name: cht-couchdb-3
+          ports:
+            - containerPort: 5984
+          resources: {}
+          {{- if eq .Values.environment "local" }}
+          volumeMounts:
+            - mountPath: /opt/couchdb/data
+              name: local-volume
+          {{- else }}
+          volumeMounts:
+            - mountPath: /opt/couchdb/data
+              name: couchdb-3-claim0
+          {{- end }}
+      restartPolicy: Always
+      {{- if eq .Values.environment "local" }}
+      volumes:
+        - name: local-volume
+          hostPath:
+            path: {{ .Values.local.diskPath }}
+      {{- else }}
+      volumes:
+        - name: couchdb-3-claim0
+          persistentVolumeClaim:
+            claimName: couchdb-3-claim0
+      {{- end }}
+status: {}
+{{- end }}
```

Comment:
This is very repetitive - is there no way to script 3 nodes so you don't have to duplicate the yaml?

Reply:
I have made the number of nodes configurable :). Defaulting to 3.